packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -27,6 +27,7 @@ def helm_handler(event, context):
     wait         = props.get('Wait', False)
     timeout      = props.get('Timeout', None)
     namespace    = props.get('Namespace', None)
+    create_namespace = props.get('CreateNamespace', None)
```

Comment:
Can you please test this with an explicit false. I suspect we might have an issue with custom resources where all the properties are stringified, so false will be treated as truthy. The solution is just to pass `undefined` instead of `false`.

[same for "Wait" by the way]

Reply:
@eladb 

You are right.

Setting it to `false` will still got `--create-namespace` in python

<img width="639" alt="圖片" src="https://user-images.githubusercontent.com/278432/86084868-78bf7a80-bad0-11ea-84e9-00a1495601b7.png">

<img width="1488" alt="圖片" src="https://user-images.githubusercontent.com/278432/86084974-a3113800-bad0-11ea-86f1-527239b56ad0.png">

Reply:
checking the `wait` parameter now.

Reply:
The solution is simple, just pass `undefined` in case the value is falsy:

```ts
const createNamespace = props.createNamespace ?? true;

// and then:
  ...
  CreateNamespace: createNamespace || undefined
```

Reply:
@eladb 

OK.

```ts
    const createNamespace = props.createNamespace ?? true;

    new CustomResource(this, 'Resource', {
        ...
        CreateNamespace: createNamespace ?? undefined,
      },
    });
```

Should we fix `wait` in this PR?

Reply:
Yes please (and add tests). I believe there is a GitHub issue that you can add as resolved. Let me dig it up

Reply:
I believe this is related https://github.com/aws/aws-cdk/issues/8713

Reply:
OK tests added. Waiting for the build result.